From 06e5861a7d74a4bf7c691c335438ebd6b21521fe Mon Sep 17 00:00:00 2001
From: "Korn, Uwe" <Uwe.Korn@blue-yonder.com>
Date: Tue, 19 Mar 2019 11:16:02 +0100
Subject: [PATCH 1/4] Deactivate aggressive failures on warnings

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1f8931508..43c486596 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -167,6 +167,8 @@ if (ENABLE_ASAN)
   endif()
 endif()
 
+set(WARN_FLAGS "")
+
 enable_testing()
 
 INCLUDE(GNUInstallDirs)  # Put it before ThirdpartyToolchain to make CMAKE_INSTALL_LIBDIR available.
