From 651672c8034c07c410aebf0e7667a6a77a31a8d7 Mon Sep 17 00:00:00 2001
From: "Korn, Uwe" <Uwe.Korn@blue-yonder.com>
Date: Tue, 19 Mar 2019 11:16:02 +0100
Subject: [PATCH 1/5] Deactivate aggressive failures on warnings

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 47a6d66..47832b5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -160,6 +160,8 @@ else ()
   add_compile_definitions(ENABLE_METRICS=0)
 endif ()
 
+set(WARN_FLAGS "")
+
 enable_testing()
 
 INCLUDE(CheckSourceCompiles)
