From c12e690560c91b7a1593b57a8b390696d73296db Mon Sep 17 00:00:00 2001
From: "Korn, Uwe" <Uwe.Korn@blue-yonder.com>
Date: Tue, 19 Mar 2019 11:16:02 +0100
Subject: [PATCH 1/5] Deactivate aggressive failures on warnings

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dcd4e7fd1..4fbda3544 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -177,6 +177,8 @@ else ()
   add_compile_definitions(ENABLE_METRICS=0)
 endif ()
 
+set(WARN_FLAGS "")
+
 enable_testing()
 
 INCLUDE(CheckSourceCompiles)
-- 
2.41.0

