diff --git a/cmake_modules/FindProtobufAlt.cmake b/cmake_modules/FindProtobufAlt.cmake
index efc004136..9292c6433 100644
--- a/cmake_modules/FindProtobufAlt.cmake
+++ b/cmake_modules/FindProtobufAlt.cmake
@@ -60,7 +60,10 @@ if (ProtobufAlt_FOUND)
     set (PROTOBUF_STATIC_LIB PROTOBUF_STATIC_LIB-NOTFOUND)
     set (PROTOC_LIBRARY protobuf::libprotoc)
     set (PROTOC_STATIC_LIB PROTOC_STATIC_LIB-NOTFOUND)
-    set (PROTOBUF_EXECUTABLE protobuf::protoc)
+    # https://github.com/protocolbuffers/protobuf/issues/14576
+    if (NOT DEFINED PROTOBUF_EXECUTABLE)
+      set (PROTOBUF_EXECUTABLE protobuf::protoc)
+    endif ()
 
     get_target_property (target_type protobuf::libprotobuf TYPE)
     if (target_type STREQUAL "STATIC_LIBRARY")
