From 7107be4b257f5ce6989b3761cc0aeece1e27be56 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 10 Jan 2025 18:08:59 +0100
Subject: [PATCH 4/4] fix capitalization of Protobuf; search with CONFIG

---
 cmake_modules/ThirdpartyToolchain.cmake | 16 ++++++++++------
 1 file changed, 10 insertions(+), 6 deletions(-)

diff --git a/cmake_modules/ThirdpartyToolchain.cmake b/cmake_modules/ThirdpartyToolchain.cmake
index 017e6c5b8..6e80ec317 100644
--- a/cmake_modules/ThirdpartyToolchain.cmake
+++ b/cmake_modules/ThirdpartyToolchain.cmake
@@ -491,7 +491,7 @@ endif ()
 
 if (ORC_PACKAGE_KIND STREQUAL "conan")
   find_package (Protobuf REQUIRED CONFIG)
-  add_resolved_library (orc_protobuf ${protobuf_LIBRARIES} ${protobuf_INCLUDE_DIR})
+  add_resolved_library (orc_protobuf ${Protobuf_LIBRARIES} ${Protobuf_INCLUDE_DIR})
   list (APPEND ORC_SYSTEM_DEPENDENCIES Protobuf)
   list (APPEND ORC_INSTALL_INTERFACE_TARGETS "$<INSTALL_INTERFACE:protobuf::libprotobuf>")
 elseif (ORC_PACKAGE_KIND STREQUAL "vcpkg")
@@ -502,18 +502,22 @@ elseif (ORC_PACKAGE_KIND STREQUAL "vcpkg")
   list (APPEND ORC_INSTALL_INTERFACE_TARGETS "$<INSTALL_INTERFACE:protobuf::libprotobuf>")
   set (PROTOBUF_EXECUTABLE protobuf::protoc)
 elseif (NOT "${PROTOBUF_HOME}" STREQUAL "")
-  find_package (Protobuf REQUIRED)
+  find_package (Protobuf CONFIG REQUIRED)
+  message(STATUS "Protobuf_LIBRARY: ${Protobuf_LIBRARY}")
+  message(STATUS "Protobuf_PROTOC_LIBRARY: ${Protobuf_PROTOC_LIBRARY}")
+  message(STATUS "Protobuf_INCLUDE_DIRS: ${Protobuf_INCLUDE_DIRS}")
+  message(STATUS "Protobuf_INCLUDE_DIR: ${Protobuf_INCLUDE_DIR}")
 
   if (ORC_PREFER_STATIC_PROTOBUF AND PROTOBUF_STATIC_LIB)
-    add_resolved_library (orc_protobuf ${PROTOBUF_STATIC_LIB} ${PROTOBUF_INCLUDE_DIR})
+    add_resolved_library (orc_protobuf ${PROTOBUF_STATIC_LIB} ${Protobuf_INCLUDE_DIR})
   else ()
-    add_resolved_library (orc_protobuf ${PROTOBUF_LIBRARY} ${PROTOBUF_INCLUDE_DIR})
+    add_resolved_library (orc_protobuf ${Protobuf_LIBRARY} ${Protobuf_INCLUDE_DIR})
   endif ()
 
   if (ORC_PREFER_STATIC_PROTOBUF AND PROTOC_STATIC_LIB)
-    add_resolved_library (orc_protoc ${PROTOC_STATIC_LIB} ${PROTOBUF_INCLUDE_DIR})
+    add_resolved_library (orc_protoc ${PROTOC_STATIC_LIB} ${Protobuf_INCLUDE_DIR})
   else ()
-    add_resolved_library (orc_protoc ${PROTOC_LIBRARY} ${PROTOBUF_INCLUDE_DIR})
+    add_resolved_library (orc_protoc ${Protobuf_PROTOC_LIBRARY} ${Protobuf_INCLUDE_DIR})
   endif ()
 
   list (APPEND ORC_SYSTEM_DEPENDENCIES Protobuf)
